README.md

```diff
@@ -45,198 +29,104 @@ The platform incorporates the following components:
 
-The top menu bar exposes available top-level actions, such as exporting a notebook in a different format.
+The platform's Jupyter Notebook service uses the JupyterLab UI, which is described in detail in the [JupyterLab documentation](https://jupyterlab.readthedocs.io/en/stable/user/interface.html#the-jupyterlab-interface).
 
-The left-sidebar menu contains commonly used tabs, including a **File Browser** (directory icon) for browsing files.<br>
-The home directory of the platform's Jupyter Notebook service contains the following files and directories:
+By default, the **File Browser** in the left-sidebar menu displays the home directory of the platform's Jupyter Notebook service, which includes these components:
 
-- **v3io** directory, which displays the contents of the `v3io` platform cluster data mount for browsing the contents of the cluster's data containers.
-  You can also browse the contents of the data containers from the **Data** page of the platform dashboard.
+- **v3io** directory, which displays the contents of the `v3io` platform cluster data mount for browsing the cluster's data containers.
+  You can also browse the containers data from the **Data** page of the platform dashboard.
 - The contents of the running-user home directory &mdash; **users/&lt;running user&gt;**.
   This directory contains the platform's [tutorial Jupyter notebooks](https://github.com/v3io/tutorials):
 
-  - **welcome.ipynb** / [**README.md**](../README.md) &mdash; the current document, which provides a short introduction to the platform and how to use it to implement a full data science workflow.
-  - [**getting-started**]() &mdash; a directory containing getting-started tutorials that explain and demonstrate how to perform different platform operations using the platform APIs and integrated tools.
-  - [**demos**](demos/README.ipynb) &mdash; a directory containing [end-to-end use-case application demos](#end-to-end-use-case-applications).
-  - Scripts and related notebooks for [updating the tutorial notebooks](#update-notebooks) and [downloading additional demo applications](#mlrun-demos-download).
+  - [**welcome.ipynb**](welcome.ipynb) / [**README.md**](../README.md) &mdash; the current document, which provides a short introduction to the platform and how to use it to implement a full data science workflow.
+  - [**platform-overview.ipynb**](platform-overview.ipynb) &mdash; an in-depth platform overview, which details how you can leverage the platform to implement each step of the data science workflow from research to production.
+  - [**getting-started**](getting-started/getting-started-basic.ipynb) &mdash; a directory containing getting-started tutorials that explain and demonstrate how to perform different platform operations using the platform APIs and integrated tools.
+  - [**demos**](demos/README.ipynb) &mdash; a directory containing [end-to-end application use-case demos](#end-to-end-use-case-applications).
```

Comment:
This was initially included in the Jupyter Notebook Basics section because it explains what they see in the file browser of the Jupyter UI. At first, we also described the tutorial directories in a previous location in the doc, but this created an unnecessary duplication. However, when editing the file recently I also thought that this doesn't really fall under Jupyter NB basics, and this specifically bothers me now that we removed the sub-section links from the main TOC. => We can consider moving this somewhere else.
We would, however, still need to refer to the **v3io/** directory in the NB basics section, because this isn't really an overview of the tutorial NBs. The doc was added to explain to users about the function of this directory, which they would inevitably see in the Jupyter home directory in the UI and might otherwise think that it contains additional tutorials. Let's discuss on the phone.

Reply:
UPDATE: See comment https://github.com/v3io/tutorials/pull/217#issuecomment-602727918 for commit bb68d18.